src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -277,6 +276,19 @@ internal BoundExpression ConvertPatternExpression(TypeSymbol inputType, CSharpSy
                                                    isVar: isVar, patternTypeWasInSource: true, diagnostics: diagnostics);
             }
 
+            // PROTOTYPE(wildcards): handle wildcard designation here
```

Comment:
Seems like it's already handled. Do we still need the PROTOTYPE marker?

Maybe simplify to:
```C#
if (node.Designation.Kind() == SyntaxKind.DiscardedDesignation)
{
    return new BoundDeclarationPattern(node, null, boundDeclType, isVar, hasErrors);
}
```